.errcheck-excluded.txt

```diff
@@ -0,0 +1,2 @@
+os.MkdirAll
```

Comment:
Would it also be possible to explicitly ignore errors in the exact locations where we want to ignore errors. Like so:
```go
_ = os.MkdirAll(...)
```

Reply:
definitely. Then we shouldn't have this file.

Reply:
yes, it's possible. For being explicit, I dropped this file.